appengine/loopback/package.json

```diff
@@ -8,19 +8,26 @@
   "engines": {
     "node": ">=4.3.2"
   },
+  "main": "server/server.js",
   "scripts": {
-    "pretest": "jshint .",
-    "start": "node server/server.js"
+    "lint": "eslint .",
+    "start": "node .",
+    "posttest": "npm run lint && nsp check"
```

Comment:
These files are generated by the loopback app scaffolder. I'm not going to manually modify any of them.